supervisor/add_process.go

```diff
@@ -34,7 +34,7 @@ func (s *Supervisor) addProcess(t *AddProcessTask) error {
 		return err
 	}
 	ExecProcessTimer.UpdateSince(start)
-	t.StartResponse <- StartResponse{}
+	t.StartResponse <- StartResponse{ExecPid: process.SystemPid()}
```

Comment:
@mlaventure if I pull this patch into Docker and expose the exec's pid I always get pid == 0; any reason? (it doesn't sound correct to me...)